core-java-modules/core-java-io-conversions-2/src/test/java/com/baeldung/arraystostring/JavaArraysToStringUnitTest.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.arraystostring;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class JavaArraysToStringUnitTest {
+
+        @Test public void givenInstanceOfArray_whenTryingToConvertToString_thenNameOfClassIsShown() {
```

Comment:
let's have the annotations in different lines than the method definition

Reply:
Let me know if this is good.